README.md

```diff
@@ -1,42 +1,62 @@
 # Flutter Templates
 
-All the templates can be used to kick off a new Flutter application quickly.
+All the templates can be used to kick off a new Flutter project quickly.
 
-## Usage
+## Features
 
-Clone the repository
+- Supports __Android__ and __iOS__ platforms *(Web and Desktop are not yet supported)*.
+- MVVM with __Clean Architecture__ and pre-built foundational components.
```

Comment:
Can we provide a good link about this `Clean Architecture`, please? 
It could help someone new like me to know what we are talking about 🙏

Reply:
@markgravity IMO, MVVM and Clean Architecture are common terms and everyone can easily find them on the internet. However, this is no official link for this, and I don't want to provide links to guides, tutorials or courses, e.g., https://www.udemy.com/course/flutter-advanced-course-clean-architecture-with-mvvm/

Reply:
Maybe I can link to https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html and

Reply:
@markgravity updated in https://github.com/nimblehq/flutter-templates/pull/238/commits/44e679c8ed299c06c17b17dd8f7266ab1300b2e4